src/components/WebexRemoteMedia/WebexRemoteMedia.scss

```diff
@@ -20,10 +20,28 @@
 
   .md-badge {
     margin: auto;
-    padding: 0.4rem;
+    height: 38px;
```

Comment:
the height should be automatically computed by the browser based on text height and paddings

Reply:
I use height because the .md-badge has a fix height which come from momentum-ui / Badge, and created same style for legacy too